src/Stores/Votes/selectors.js

```diff
@@ -1,3 +1,14 @@
+/**
```

Comment:
seems we're missing the `@flow` at the top of this file.
Could you add it?

Reply:
Its been added